Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -34,7 +34,12 @@ @interface ARMapContainerViewController () <PulleyDelegate, PulleyDrawerViewCont
 {
     for (UIView *subview in view.subviews) {
         if ([subview isKindOfClass:UIScrollView.class]) {
-            return (UIScrollView *)subview;
+            CGSize size = [(UIScrollView *)subview contentSize];
+             // The tab bar on the City guide is a scrollview, so we need to make sure we hit
+            //  the main scrollview instead.
+            if (size.height > size.width ) {
+                return (UIScrollView *)subview;
+            }
```

Comment:
The name of this function doesn’t reflect the change, it should probably be called `FindMapScrollView` or something. I would also prefer to look for known classes to bail on, e.g. the main map view class, rather than this ambiguous check.

Reply:
👍 I'll update the naming in a subsequent PR